src/coreclr/hosts/corerun/corerun.cpp

```diff
@@ -33,7 +33,7 @@ static const wchar_t *coreCLRDll = W("CoreCLR.dll");
 static const wchar_t *coreCLRInstallDirectory = W("%windir%\\system32\\");
 
 // Encapsulates the environment that CoreCLR will run in, including the TPALIST
-class HostEnvironment 
```

Comment:
This file doesn't have any modifications outside of whitespace changes.

Reply:
It is dirty commit - just needed to run CI - everything will get cleaned later

Reply:
You should be able to comment `@dotnet-bot test <test-name> please` to rerun any given job, use `this` to rerun all jobs